Sources/web3swift/Operations/WriteOperation.swift

```diff
@@ -15,21 +15,26 @@ public class WriteOperation: ReadOperation {
     /// - Parameters:
     ///   - password: Password for private key.
     ///   - policies: Custom policies for how to resolve (optional). Default is auto.
-    public func writeToChain(password: String, policies: Policies = .auto) async throws -> TransactionSendingResult {
+    public func writeToChain(password: String, policies: Policies = .auto, sendRaw: Bool = false) async throws -> TransactionSendingResult {
```

Comment:
Could we made it true by default, since it's obvious more often use case as false one? I guess this leads to test failing, maybe we could enhance it here as well?

Also, just thoughts for future PRs. I thing it would be much much more convenient for both of us to split such PRs of each other. Like make one that getting code prettier, but not enhance its logic in any way, and second with logic enhancing.

Also, it could be applicable here as well. Like we still have a possibility to drop exact this bit in separate PR covered by tests, and maybe we could even pass it before the whole cleanup. What would you say about that @pharms-eth?

Reply:
yes setting it true by default makes the tests fail

I will consider, I worked on a feature in my projects and these were the enhancements that occurred over that period.

The Code is combines now, so submitting different PRs at this stage is not straight forward @yaroslavyaroslav It may be easier to redo everything at that point which is not in my interest